.gitignore

```diff
@@ -18,3 +18,4 @@ testem.log
 lcov.dat
 sauce-example.log
 .env
+.tm_properties
```

Comment:
We don't have any other IDEs in here. @stopstedt thought on including specific ignore stuff?